.gitignore

```diff
@@ -17,3 +17,6 @@ internal/util/version/branch.txt
 **/testdata/fuzz
 
 docker-compose.override.yml
+go.wor
+go.wor.sum
+tst.txt
```

Comment:
could you please remove the private file which probably won't appear in another person's local file directory,  from the gitignore?